### ISSUE TITLE
build system RRV_DEBUG defaults to ON

### ISSUE THREAD

**Issue by claudeha** [kdudka/rrv#3]
Causing extreme slowness.  Not documented in the README.

Using:

```
cmake -DRRV_DEBUG=OFF .
```

sped up the `room2.xml` example from around 5.7 patches per second to around 32.3 patches per second on my system.

**Comment by claudeha** [comment]
also setting it to OFF needs `size_t` (for example from `#include <cstddef>`) in

```
   src/PatchCacheLine.h
   src/TriangleSet.h
   src/TriangleSetExt.h
```

**Comment by kdudka** [comment]
Good catch!  Fixed via 1cecfd2c.
